lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-zoom-group' }, () => {
+          this.a({ outlet: 'zoomInButton', class: 'image-controls-zoom-in' }, () => {
+            this.text('+')
+          })
+          this.a({ outlet: 'zoomOutButton', class: 'image-controls-zoom-out' }, () => {
+            this.text('-')
+          })
+          this.a({ outlet: 'zoomResetButton', class: 'image-controls-zoom-reset' }, () => {
+            this.text('1:1')
+          })
+        })
```

Comment:
The markup should be like in the origin Image view. See: [atom/image-view -> lib/image-editor-view.coffee](https://github.com/atom/image-view/blob/master/lib/image-editor-view.coffee#L20)